pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -761,7 +761,9 @@ stdenv.mkDerivation (finalAttrs: {
 
   strictDeps = true;
 
-  nativeBuildInputs = [ removeReferencesTo addDriverRunpath perl pkg-config texinfo yasm ]
+  nativeBuildInputs = [ removeReferencesTo addDriverRunpath perl pkg-config yasm ]
+  # Texinfo version 7 introduced breaking changes, which older versions of ffmpeg do not handle.
```

Comment:
To my knowledge Texinfo version 7 already introduced breaking changes in their Perl API, what we are seeing here with Texinfo version 7.1 and `ffmpeg_4` specifically appears to be just a continuation of that.
If you see value in being more specific here, I'll happily amend the version, @alyssais 🙂

Reply:
Since this is the ffmpeg package, and ffmpeg worked fine with 7.1 (right?), I think 7.1 should be the version mentioned.

Reply:
[Please hold](https://music.youtube.com/watch?v=k7_KNTF-QoI) while I double check; be right with you… 🙂

Reply:
Wasn't it that it was deprecated in 7 and they then removed it in 7.1?

Reply:
Yes, @alyssais, building the `ffmpeg` package worked and works fine with texinfo 7.1.

By "it", do you mean the Perl API, @Atemu ? If yes, at first glance it appears to be quite alive to me, see https://www.gnu.org/software/texinfo/manual/texi2any_api/texi2any_api.html

Reply:
Sorry, I meant the breaking change. IIRC something was deprecated in 7.0 and then removed/broken in 7.1.

Reply:
Thanks for the clarification, @Atemu, I haven't found a definite answer on that (yet).

What is left to do on this PR in order to move forward? 🙂 
Just the comment change requested above?

Reply:
Change the comment to

```suggestion
  # Texinfo version 7.1 introduced breaking changes, which older versions of ffmpeg do not handle.
```

because that's where it broke. From our perspective, it doesn't really matter when it was deprecated, just when it broke.

Reply:
Done ✅ 🙂